Vungle/VungleBannerCustomEvent.m

```diff
@@ -156,7 +137,6 @@ - (void)vungleAdDidFailToLoad:(NSError *)error
 
 - (void)vungleAdDidFailToPlay:(NSError *)error
 {
-    // verify - is this necessary?
```

Comment:
I didn't find Dravya’s definite comments whether need to remove vungleAdDidFailToPlay: function, so according to Aki's comment, I remove the comment at least. :)

Reply:
@YueVungle I think you can remove this callback as well since we already sent `[self.delegate bannerCustomEvent:self didLoadAd:mrecAdView];` in `- (void)vungleAdDidLoad`.

Reply:
@ashinagawa  Got it, done. :)